spec/factories/metadata/school_contract_period_factory.rb

```diff
@@ -4,7 +4,7 @@
     association :contract_period
 
     updated_at { Faker::Time.between(from: 1.month.ago, to: Time.zone.now) }
-    in_partnership { Faker::Boolean.boolean }
-    induction_programme_choice { Metadata::SchoolContractPeriod.induction_programme_choices.keys.sample }
+    in_partnership { true }
```

Comment:
can we keep those random and in the tests set them specifically instead? as this will affect seed data generation

Reply:
Ah I forgot we introduced them to the seed data 🤔 in my experience random factories == falkiness, but I'll revert it